compiler/souschef/src/Gaussian.cpp

```diff
@@ -50,22 +50,26 @@ std::vector<uint8_t> GaussianFloat32DataChef::generate(int32_t count) const
   return res;
 }
 
-std::vector<uint8_t> GaussianInt32DataChef::generate(int32_t count) const
+template <typename T> std::vector<uint8_t> GenerateGaussian(int32_t count, float mean, float stddev)
```

Comment:
could you please split this change (maybe this change first before adding int16) to another PR?
this seems a bit different context.

Reply:
Do you mean split template introduction from actual support?

Reply:
Yup :)